samples/Tests/BackgroundImage.VerticalAndHorizontalAlignment.json

```diff
@@ -114,7 +114,7 @@
 				},
 				{
 					"type": "Column",
-					"width": "auto",
```

Comment:
>th": "auto", [](start = 9, length = 12)

do we have other tests that cover the `"width": "auto"` case?

Reply:
yes but the result looks better with stretch width as auto overflows the card in UWP and WPF

---
In reply to: [286154178](https://github.com/microsoft/AdaptiveCards/pull/2955#discussion_r286154178) [](ancestors = 286154178)

Reply:
that's fine -- I just wanted to make sure we were getting visual verification for the `auto` case :)

---
In reply to: [286155401](https://github.com/microsoft/AdaptiveCards/pull/2955#discussion_r286155401) [](ancestors = 286155401,286154178)